### ISSUE TITLE
project template enhancement and updates

### ISSUE THREAD

**Issue by rgbayrak** [brainhack-vandy/brainhack-vandy.github.io#49]
Update project template with BHVU specific details
- [x] long title -> short title to be parsed for projection to the website
- [x] replace  (and Twitter / Mattermost handle if possible)  with  (and GitHub handle if possible)  
- [x] remove questions from project desc "- How to get started? Where to find key resources?"
- [x] remove section on communication channels
- [x] make "Data to use" required field
- [x] Credit to collaborators > All project contributors will be acknowledged through:
 (1) GitHub repository README.md with a clear contributors section listing all team members and their roles, for example:
(2) If the project results in any publications or presentations, all contributors will be listed as co-authors following their level of involvement.
- [x] Global to Vanderbilt

@dgzct11 
- [x] Flip what is being parsed for the project title
- [ ] enable project-monitors or remove the redundant info from template
- [x] remove github link, add Project button (pointing at the project pitch issue), (optionally tho not necessary) add Repo button (pointing at the original repo if exists)
- [x] project image is not showing

**Comment by rgbayrak** [comment]
@dgzct11 any progress on project-monitors?

**Comment by dgzct11** [comment]
Yes, I found the documentation for creating a team within a repo and enabling notifications: [https://docs.github.com/en/organizations/organizing-members-into-teams/configuring-team-notifications](url). this page describes the creation of a team: [https://docs.github.com/en/organizations/organizing-members-into-teams/creating-a-team](url). However it requires permissions within the brain hack vandy organization, which I don't have.
